htap.go

```diff
@@ -0,0 +1,8 @@
+package gosnowflake
+
+type queryContextEntry struct {
```

Comment:
you are adding one struct but PR says that there will be struct**s** - are you going to add more of them in this PR?

Reply:
No, that's it. Changed commit msg.

Reply:
Let's change also the PR name